index.bs

```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
 ============
 
 The OrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
-to provide information that describes the device's physical orientation.
+to provide generic information describing the device's physical orientation
+in relation to a three dimensional Cartesian coordinate system.
 
-The base object describes the orientation in relation to <a>Earth's reference
-coordinate system</a>, thus in a stationary three dimensional Cartesian
-coordinate system.
+The AbsoluteOrientationSensor API inherits from the OrientationSensor API and
```

Comment:
More clear and consistent with links: {{AbsoluteOrientationSensor}} {{OrientationSensor}}